src/server.ts

```diff
@@ -140,17 +141,98 @@ const statusRequest = async (
   }
 };
 
-/* Redirects to user profile when linked.
-   We don't do any fancy special embeds yet, just Twitter default embeds. */
-const profileRequest = async (request: IRequest) => {
-  const { handle } = request.params;
+/* Handler for User Profiles */
+const profileRequest = async (request: IRequest, event: FetchEvent,
+  flags: InputFlags = {}) => {
+  const { handle, language } = request.params;
   const url = new URL(request.url);
+  const userAgent = request.headers.get('User-Agent') || '';
+
+  /* User Agent matching for embed generators, bots, crawlers, and other automated
+     tools. It's pretty all-encompassing. Note that Firefox/92 is in here because 
+     Discord sometimes uses the following UA:
+     
+     Mozilla/5.0 (Macintosh; Intel Mac OS X 11.6; rv:92.0) Gecko/20100101 Firefox/92.0
+     
+     I'm not sure why that specific one, it's pretty weird, but this edge case ensures
+     stuff keeps working.
+     
+     On the very rare off chance someone happens to be using specifically Firefox 92,
+     the http-equiv="refresh" meta tag will ensure an actual human is sent to the destination. */
+  const isBotUA = userAgent.match(Constants.BOT_UA_REGEX) !== null;
 
   /* If not a valid screen name, we redirect to project GitHub */
   if (handle.match(/\w{1,15}/gi)?.[0] !== handle) {
     return Response.redirect(Constants.REDIRECT_URL, 302);
+  }
+  const username = handle.match(/\w{1,15}/gi)?.[0] as string;
+  /* Check if request is to api.fxtwitter.com, or the tweet is appended with .json
+     Note that unlike TwitFix, FixTweet will never generate embeds for .json, and
+     in fact we only support .json because it's what people using TwitFix API would
+     be used to.
+  */
+  if (
+    url.pathname.match(/\/status(es)?\/\d{2,20}\.(json)/g) !== null ||
```

Comment:
Remove this regex, it's only relevant for statuses with the legacy TwitFix endpoint.

Reply:
Also the comment above as again it's not relevant for profiles